src/backend/core/authentication/backends.py

```diff
@@ -88,6 +90,11 @@ def get_or_create_user(self, access_token, id_token, payload):
         sub = user_info["sub"]
         email = user_info.get("email")
 
+        if settings.FEATURES_ALPHA:
+            alpha_enabled = posthog.feature_enabled("alpha", email)
+            if not alpha_enabled:
```

Comment:
```suggestion
            if not posthog.feature_enabled("alpha", email):
```

Reply:
It is on purpose that we don't want to make a Posthog feature flag call if `FEATURES_ALPHA` is not set. ( It is optional )

Reply:
With my suggestion you will still fetch posthog only if the settings `FEATURES_ALPHA` is set to True.

Reply:
Read to quick, k